### PR TITLE
Enable the auto-cc bot

### LINKED ISSUE #9216: Issue Label Notifications by using GitHub Action
### Motivation

As PyTorch Lightning Community grows, some users want to be notified by issues with specific labels. By using GitHub Action would allow to notify the interested users. 
Basically, the GitHub Action runs every time a label is attached to an issue, and compares the label to the list of notification recipients that has been specified If a match is found, the action will add a comment to the issue and @mention the notification recipients.

### Pitch

Introduce Issue label notifier by using GitHub Action. 
https://github.com/marketplace/actions/issue-label-notifier

### Alternatives
Keep it as it is. 
Use the UI search tool to find the issues/PR with corresponding labels. 

### Additional context

Part of https://github.com/PyTorchLightning/pytorch-lightning/issues/9140

______________________________________________________________________

#### If you enjoy Lightning, check out our other projects! ⚡

<sub>

- [**Metrics**](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.

- [**Flash**](https://github.com/PyTorchLightning/lightning-flash): The fastest way to get a Lightning baseline! A collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning

- [**Bolts**](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks and more for research and production with PyTorch Lightning and PyTorch

- [**Lightning Transformers**](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.

</sub>

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

Fixes #9216
Uses #10530 

### Does your PR introduce any breaking changes? If yes, please list them.

None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

### CODE REVIEW

File: .github/workflows/probot-auto-cc.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  auto-cc:
+    if: ${{ github.repository_owner == 'PyTorchLightning' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: carmocca/probot@v1
```
Comment (carmocca) [inline .github/workflows/probot-auto-cc.yml:16]: I need an admin to publish the action, until then, we can use my own

File: .github/lightning-probot.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+tracking_issue: 10530
```
Comment (carmocca) [inline .github/lightning-probot.yml:1]: Tested in my fork. Seems to be working

But this file needs to be merged to master.

Comment (akihironitta) [reply .github/lightning-probot.yml:1]: https://github.com/carmocca/pytorch-lightning/issues/4 :tada:

**Review by akihironitta — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by justusschock — APPROVED** [review]

File: .github/workflows/probot-auto-cc.yml (hunk 1 above)
Comment (Borda) [reply .github/workflows/probot-auto-cc.yml:16]: have you asked for it? I do not see it anywhere in communication...
